### PR TITLE
Fixes #3179: Optimised reading_text_size_fragment implementation

### LINKED ISSUE #3179: Optimise reading_text_size_fragment.xml implementation
We should shift the `reading_list_app_bar_layout` from `mobile` xml files to the `reading_text_size_activity` so that we can remove the `layout-sw600dp/reading_text_size_fragment`.

### PR DESCRIPTION
**Pull request by bhaktideshmukh**
## Explanation
Fixes #3179 

- Shifted reading_list_app_bar_layout from mobile xml files to the reading_text_size_activity.
- Deleted layout-sw600dp/reading_text_size_fragment file.
 
**Screeshots**

Phone portrait mode

| Before  | After|
|-------|---|
|||

Phone landscape mode
| Before  | After|
|-------|---|
|||

Tablet portrait mode

| Before  | After|
|-------|---|
|||

Tablet landscape mode

| Before  | After|
|-------|---|
|||

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by bhaktideshmukh** [comment]
@rt4914 
Gentle reminder to review this PR.
Thanks.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@bhaktideshmukh Please add tablet screenshots too even if they are part of fragment.

File: app/src/main/res/layout/reading_text_size_activity.xml [hunk 1]
```diff
@@ -1,5 +1,52 @@
...
+    <View
+      android:id="@+id/toolbar_shadow_view"
+      android:layout_width="match_parent"
+      android:layout_height="6dp"
+      android:background="@drawable/toolbar_drop_shadow"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/reading_list_app_bar_layout"/>
+  </androidx.constraintlayout.widget.ConstraintLayout>
+</layout>
```
Comment (rt4914) [inline app/src/main/res/layout/reading_text_size_activity.xml]: EOF is missing, add one blank line at the end.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@bhaktideshmukh PTAL

File: app/src/main/res/layout/reading_text_size_activity.xml (hunk 1 above)
Comment (bhaktideshmukh) [reply app/src/main/res/layout/reading_text_size_activity.xml]: Done

**Comment by bhaktideshmukh** [comment]
> @bhaktideshmukh Please add tablet screenshots too even if they are part of fragment.

Done

**Comment by BenHenning** [comment]
> > @bhaktideshmukh Please add tablet screenshots too even if they are part of fragment.
> 
> Done

@bhaktideshmukh as far as I can tell the test failures seem new in this PR. I think you'll need to investigate this. I suggest:
- Verifying that the broken test suite passes locally for you on the develop branch (i.e. without your change)
- Verifying that the broken test suite correctly fails locally for you when run on your feature branch (I suggest running it a few times to verify that it's consistent)
- Reverting the PR locally line-by-line or test-by-test for common files (i.e. the manifest, the test activity, and the broken test itself) until it no longer fails; that'll help you isolate whatever change caused it
- Investigating into the offending change to determine how it affects the broken test (which should help you root cause the problem)

**Comment by BenHenning** [comment]
Apologies. I will need to look at this tomorrow.

File: app/src/main/res/layout/reading_text_size_activity.xml [hunk 2]
```diff
@@ -1,5 +1,49 @@
...
+  <View
+    android:id="@+id/toolbar_shadow_view"
+    android:layout_width="match_parent"
+    android:layout_height="6dp"
+    android:background="@drawable/toolbar_drop_shadow"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toBottomOf="@id/reading_list_app_bar_layout" />
+</androidx.constraintlayout.widget.ConstraintLayout>
+
```
Comment (rt4914) [inline app/src/main/res/layout/reading_text_size_activity.xml]: Remove this extra blank line

**Review by rt4914 — CHANGES_REQUESTED** [review]
@bhaktideshmukh just minor change

File: app/src/main/res/layout/reading_text_size_activity.xml (hunk 2 above)
Comment (bhaktideshmukh) [reply app/src/main/res/layout/reading_text_size_activity.xml]: Done

**Review by rt4914 — APPROVED** [review]
LGTM

**Pull request merged**